### PR TITLE
Refactoring test utils.

### PR DESCRIPTION
**Pull request by DamianMullins**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "babel-preset-babili": "^0.1.4",
     "babel-preset-es2015": "^6.24.1",
     "gulp": "^3.9.1",
+    "js-test-buddy": "^0.0.3",
```
Comment (ashleynolan) [inline package.json]: Is this something anyone can extend (or will need to extend), or do you imagine it not being needed to?

Comment (DamianMullins) [reply package.json]: Likely won't need to change very often. It's something I built so we can submit a PR to make updates, or even fork in the future if we need to.

Comment (ashleynolan) [reply package.json]: Yeh, as long as there’s a way to update if you go away on holiday or get hit by a bus I’m happy (not about you being hit by a bus in that scenario obviously… 😱 )

Comment (DamianMullins) [reply package.json]: I've added you, Simon, and Nick as collaborators on the projects.

**Pull request merged**